uber/config.py

```diff
@@ -106,18 +106,18 @@ def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
-            if dt or c.HARDCORE_OPTIMIZATIONS_ENABLED:
+            for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
+                if (dt or sa.localized_now()) >= day:
+                    price = bumped_price
+                    # If we set a price during the event, it should be used regardless of badge sales
+                    if c.EPOCH >= day >= c.ESCHATON:
```

Comment:
Ok, other PR is open at #2278, which looks slightly different than what I described above because I saw a simpler way to handle it.